src/sql/workbench/contrib/query/browser/actions.ts

```diff
@@ -52,6 +52,10 @@ export class SaveResultAction extends Action {
 	public static SAVEJSON_LABEL = localize('saveAsJson', "Save As JSON");
 	public static SAVEJSON_ICON = 'saveJson';
 
+	public static SAVEMARKDOWN_ID = 'grid.saveAsMarkdown';
+	public static SAVEMARKDOWN_LABEL = localize('saveAsMarkdown', "Save As Markdown");
+	public static SAVEMARKDOWN_ICON = 'saveCsv';
```

Comment:
Should use a separate icon for this

Reply:
Agreed - though I don't know what the designers would prefer. I added one based on the public domain markdown logo and tried to follow whatever guidelines were set for the other export icons, but I have no way of knowing if they fit the bill.

Reply:
We can use the generic MD one for now and follow up to see if design has another preference.